index.js

```diff
@@ -133,69 +139,20 @@ client.on('message', (msg) => {
                     type: 'minecraft',
                     host: 'mc.datafire.dev', //or whatever server link it is..
                 }).then((state) => {
-                    msg.channel.send({
-                        embed: {
-                            color: 3066993,
-                            author: {
-                                name: "Minecraft",
-                                icon_url: "http://www.rw-designer.com/icon-image/5547-64x64x4.png",
-                            },
-                            title: "Mint Ice Cream",
-
-                            description: "hosted by ice cream team.",
-                            fields: [{
-                                name: "Server Info",
-                                value: (`**${state.connect}**`),
-                            }, {
-                                name: "Players",
-                                value: (`**${state.players.length}**`),
-                            }, {
-                                name: "Max Players",
-                                value: (`**${state.maxplayers}**`),
-                            }],
-
-                            footer: {
-                                icon_url: client.user.avatarURL,
-
-          
-                            }
-                        }
+                    msg.channel.send({embed: {color: 3066993,author: {name: "Minecraft",icon_url: "http://www.rw-designer.com/icon-image/5547-64x64x4.png",},title: state.name ,description: "Hosted by the Ice Cream Team.",fields: [{name: "Server Info",value: (`**${state.connect}**`),}, {name: "Players",value: (`**${state.players.length}**`),}, {name: "Max Players",value: (`**${state.maxplayers}**`),}],footer: {icon_url: client.user.avatarURL,}}});

```

Comment:
Hmmm. This doesn't matter too much, but I feel as though it adds more clarity when we have the embed stuff on different lines, and it makes merging easier if someone needs to change a portion of the embed. I get that it will increase the number of lines of the application by a lot.